lib/mailer_service/last_seen_last_sync_service.rb

```diff
@@ -21,7 +21,7 @@ def self.processed_data
               # Count the total sites (sum of all sites across all districts)
               total_sites += 1
       
-              if site['days_since_last_activity'].to_i > 3 && site['days_since_last_seen'].to_i == 0
+              if site['days_since_last_activity'].to_i > 3
```

Comment:
This will cause the site to appear in both when the last seen is greater than 3 days